### ISSUE TITLE
Website [Expressive theme sunset] Update Designing page

### ISSUE THREAD

**Issue by janchild** [carbon-design-system/carbon-for-ibm-dotcom-website#1084]
We'll need to review the content on the [Designing page](https://www.ibm.com/standards/web/carbon-for-ibm-dotcom/designing/) to capture the latest information about the kits and type styles etc. 

Note from Jan: I've tightened up this page to create a narrative flow for a designer coming in and getting set up the first time, or coming back to find new information. 

There is [another ticket](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues/6445) detailing future changes to the Designing page, possibly breaking the content into separate pages but this would be better placed after we have some new steps to take for kits and new resources. 

The PR for this ticket is [here](https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/pull/1153) and ready to review.

**Comment by janchild** [comment]
I've tightened up this page to create more of a flow for a designer coming in and getting set up the first time, or coming back to find new information. 

I reviewed this content in light of [this ticket](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues/6445) that details future changes to the Designing page, possibly breaking the content into separate pages but this break out work would be better placed after we have some new steps to take for kits and new resources. 

The PR for this ticket is [here](https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/pull/1153) and is ready to review.
